aws/ecs/task_execution_policy_documents.tf

```diff
@@ -121,3 +121,16 @@ data "aws_iam_policy_document" "etl_policies" {
   }
 
 }
+
+data "aws_iam_policy_document" "send_alerts_to_CloudWatch_ecs_task" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogStreams"
+    ]
+    resources = ["arn:aws:logs:*:*:*"]
```

Comment:
Is it possible to limit this only the log groups that it needs to write to? 

Also can we leverage the statement here: https://github.com/cds-snc/covid-alert-metrics-terraform/blob/47cc7ca54e6c7d7799d0bc505e5a1dcca2490b6b/aws/ecs/task_execution_policy_documents.tf#L52

And just a separate logs:DescribeLogStreams statement?

Reply:
Good call. Let me look into creating a log group ahead of time and then reference that here. Right now its setup to auto create which is why i went with the *

Reply:
All falco docs seem to reference the same policy resource of ""arn:aws:logs:*:*:*"", so i won't know ahead of time what it's writing to since it creates dynamic log streams based on the source, so this may have to stay this way for now until i start seeing logs generated.

Reply:
What about a region and account lockdown like so?
```terraform
arn:aws:logs:${var.region}:${var.account_id}:log-group:*
```
And then once we see how it names the log streams, it could get even tighter:
```terraform
arn:aws:logs:${var.region}:${var.account_id}:log-group:someFalcoPrefixIfWeAreLucky*
```

Reply:
added, thanks for the suggestion!